README.md

```diff
@@ -23,7 +23,48 @@ Alternatively, if you want the bleeding edge version of the package you can inst
 
 ### Dependencies
 
-Bambi requires working versions of numpy, pandas, matplotlib, patsy, pymc3, and theano. Dependencies are listed in `requirements.txt`, and should all be installed by the Bambi installer; no further action should be required.
+Bambi requires working versions of arviz, formulae, numpy, pandas, pymc3, and statsmodels. Dependencies are listed in `requirements.txt`, and should all be installed by the Bambi installer; no further action should be required.
+
+## Example
+
+In the following two examples we assume the following basic setup
+
+```python
+import bambi as bmb
+import numpy as np
+import pandas as pd
+
+data = pd.DataFrame({
+    "y": np.random.normal(size=50),
+    "g": np.random.choice(["Yes", "No"], size=50)
+    "x1": np.random.normal(size=50),
+    "x2": np.random.normal(size=50)
+})
+```
+
+### Linear regression
+
+```python
+model = bmb.Model("y ~ x1 + x2 + x3", data)
+fitted = model.fit()
+```
+
+In the first line we create and build a Bambi `Model`. The second line tells the sampler to start
+running and it returns an `InferenceData` object, which can be passed to several ArviZ functions
+such as `az.summary()` to get a summary of model parameters posterior or `az.plot_traces()` to
+visualize them.
+
+### Logistic regression
+
+Here we just add the `family` argument set to `"bernoulli"` to tell Bambi we are modelling a binary
+response. By default, it uses a logit link. We can also use some syntax sugar to specify which event
+we want to model. We just say `g['Yes']` and Bambi will understand we want to model the probability
+of a `"Yes"` response. Bambi will pick a value and inform about it if we don't use this notation.
```

Comment:
"Bambi will pick a value and inform about it if we don't use this notation." not very clear